### DISCUSSION TITLE
Feature Request/Clarification: Create annotations using Google Logs-based Metrics as a data source.

### DISCUSSION THREAD

**Discussion by jonathanaylward** [grafana/grafana#41177] [Data Sources]
The documentation here seems to suggest that only custom-metrics work with grafana annotations, yet it is possible to choose logs-based metrics from withing the Annotation creation UI.  This is a bit confusing, but I get it.  Programming is hard.  Regardless, It would be nice if we could use data from logs-based metrics for annotations, without needing a logs aggregator like Loki. 

https://grafana.com/docs/grafana/latest/datasources/google-cloud-monitoring/

Specific Example:
I created a google logs-based metric based on a log that is created every time a gitlab job is run. I tried to create an annotation in a given dashboard using the metric.  The annotation query does return data, but grafana seems unable to parse the data format

`dataframes | [ "QVJST1cxAAD/////.............`

**Comment by sunker** [comment]
Thanks for reporting this @jonathanaylward. I'll convert this issue into a github discussion since that's what we use for feature requests.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
